docs/user-guide/custom-errors.md

```diff
@@ -1,6 +1,6 @@
 # Custom errors
 
-In case of an error in a request the body of the response is obtained from the `default backend`.
+In case of an error in a request, the body of the response is obtained from the `default backend`.
```

Comment:
Might be worth rephrasing if the sentence is unclear.
Suggestion: "In case the request yields an HTTP error, the body of the response will be served from the default backend."